Example/ViewController.swift

```diff
@@ -240,6 +240,11 @@ class ViewController: UIViewController {
         waypoints.insert(userWaypoint, at: 0)
 
         let options = NavigationRouteOptions(waypoints: waypoints)
+        
+        // Get periodic updates regarding changes in estimated arrival time and traffic congestion segments along the route line.
+        options.profileIdentifier = .automobileAvoidingTraffic
+        options.refreshingEnabled = true
```

Comment:
Per https://github.com/mapbox/mapbox-directions-swift/pull/420#discussion_r483287478, NavigationRouteOptions should set this value by default.